### ISSUE TITLE
Add support for test glob files

### ISSUE THREAD

**Issue by ianlnf** [nearform-actions/github-action-test-compare#31]
The current `tests` input supports a root folder only. This could be updated to support a comma-separated list of globs that are copied to the target folder keeping the same structure. E.g. `tests: __tests__,**/*.test.*`

**Comment by alcidesqueiroz** [comment]
I spent some hours working on this issue and I wanted to share some findings. Unfortunately, I wasn't able to finish it within the time box I had set for myself, so I had to move on to another task. But here's what I found:

When I ran the tests, all of them were failing both locally and on my private copy of this repo due to some expected steps not being returned from the event result. After investigating, I found that lines 42-49 of the `createMockGitHub` utility function were creating the `__target__` directory, which caused the action to fail on the step `Fail if target folder exists`. Removing those lines fixed the issue.

I also tried two approaches to implement glob support: one bash-based, and the other using the nearform-actions/github-action-copy-files. While both approaches are feasible, I wasn't able to make them work properly within my time box.
